### PR TITLE
Add modeless wrench config option

### PR DESCRIPTION
**Pull request by DeltaNedas**
ye

### CODE REVIEW

File: src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityGiantPropellerPart.java [hunk 1]
```diff
@@ -142,6 +146,21 @@ public void update() {
         }
     }
 
+    @Override
+    public boolean attemptToAssembleMultiblock(World worldIn, BlockPos pos, EnumFacing facing) {
+        List<IMultiblockSchematic> schematics = MultiblockRegistry
+        .getSchematicsWithPrefix("multiblock_giant_propeller");
```
Comment (DaMatrix) [inline src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityGiantPropellerPart.java]: any particular reason for newline here?

File: src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityGiantPropellerPart.java [hunk 2]
```diff
@@ -142,6 +146,21 @@ public void update() {
...
+        for (IMultiblockSchematic schematic : schematics) {
+            GiantPropellerMultiblockSchematic propSchem = (GiantPropellerMultiblockSchematic) schematic;
+            if (propSchem.getPropellerFacing() == facing) {
+                if (schematic.attemptToCreateMultiblock(worldIn, pos)) {
```
Comment (DaMatrix) [inline src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityGiantPropellerPart.java]: y not `&&`

File: src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityRudderPart.java [hunk 3]
```diff
@@ -221,4 +224,20 @@ public AxisAlignedBB getRenderBoundingBox() {
         }
     }
 
+    @Override
+    public boolean attemptToAssembleMultiblock(World worldIn, BlockPos pos, EnumFacing facing) {
+        List<IMultiblockSchematic> schematics = MultiblockRegistry
+            .getSchematicsWithPrefix("multiblock_rudder_axle");
```
Comment (DaMatrix) [inline src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityRudderPart.java]: newline

File: src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityRudderPart.java [hunk 4]
```diff
@@ -221,4 +224,20 @@ public AxisAlignedBB getRenderBoundingBox() {
...
+        for (IMultiblockSchematic schematic : schematics) {
+            RudderAxleMultiblockSchematic rudderSchem = (RudderAxleMultiblockSchematic) schematic;
+            if (facing.getAxis() != rudderSchem.getAxleAxisDirection().getAxis()) {
+                if (rudderSchem.getAxleFacingDirection() == facing) {
+                    if (schematic.attemptToCreateMultiblock(worldIn, pos)) {
```
Comment (DaMatrix) [inline src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityRudderPart.java]: `&&`

File: src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityValkyriumCompressorPart.java [hunk 5]
```diff
@@ -87,4 +92,15 @@ public double getCurrentKeyframe(double partialTick) {
         return prevKeyframe + (increment * partialTick) + 1;
     }
 
+    @Override
+    public boolean attemptToAssembleMultiblock(World worldIn, BlockPos pos, EnumFacing facing) {
+        List<IMultiblockSchematic> valkyriumEngineMultiblockSchematics = MultiblockRegistry
+        .getSchematicsWithPrefix("multiblock_valkyrium_compressor");
```
Comment (DaMatrix) [inline src/main/java/org/valkyrienskies/addon/control/block/multiblocks/TileEntityValkyriumCompressorPart.java]: newline

**Comment by DeltaNedas** [comment]
tabs -> spaces because i use tabs and dont want to start a shitstorm so i convert them after im done
newline because its like that in old wrench source
&& because idk ill fix it

**Comment by DaMatrix** [comment]
idk, tabs seemed to have changed in a lot of totally random places (just asking for possible merge conflicts, code style changes should be separate from everything else unless it's in code you're already modifying anyway)

**Comment by DeltaNedas** [comment]
ok fixed it

**Pull request merged**